README.md

```diff
@@ -27,12 +27,12 @@ Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/S
  * LINUX (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
  * Apache2
  * MySQL 5.7 
- * PHP <b>7</b>  
+ * PHP <b>7.1.3</b>  
```

Comment:
See https://github.com/aces/Loris/pull/3712

Reply:
We should be telling people to install 7.2, not 7.1